deployment/cfn/worker.py

```diff
@@ -240,6 +248,10 @@ def get_cloud_config(self):
         return ['#cloud-config\n',
                 '\n',
                 'write_files:\n',
+                '  - path: /etc/rf.d/env/RF_STACK_COLOR\n',
```

Comment:
It comes from the command line argument to `rf_stacks.py`. That gets threaded into the `GlobalConfigNode`, which feeds the stack's parameters. Once that makes it into CloudFormation land, it gets templated into the `cloud-config` for a given stack, which writes the color to `/etc/rf.d/env/RF_STACK_COLOR`.